jvm/ml4ir-inference/src/main/scala/ml4ir/inference/tensorflow/data/TFRecordBuilders.scala

```diff
@@ -21,17 +40,20 @@ class SequenceExampleBuilder[C, S](contextFeaturesPreprocessor: FeaturePreproces
     * @param docs array of document-feature structs
     * @return TensorFlow's protobuf structure containing the raw features in one SequenceExample packet
     */
-  def fromExamples(context: Example, docs: Array[Example]): SequenceExample = {
-    val contextFeatures: Features = buildMultiFeatures(context.features)
-    val docFeatures: FeatureLists = buildMultiFeatureLists(docs.map(_.features))
+  def fromMultiFeatures(context: MultiFeatures, docs: Array[MultiFeatures]): SequenceExample = {
+    val contextFeatures: Features = buildMultiFeatures(context)
+    val docFeatures: FeatureLists = buildMultiFeatureLists(docs)
     SequenceExample
       .newBuilder()
       .setContext(contextFeatures)
       .setFeatureLists(docFeatures)
       .build()
   }
+}
+
+trait TFRecordBuilderUtils {
 
-  private def buildMultiFeatures(features: MultiFeatures): Features = {
+  protected def buildMultiFeatures(features: MultiFeatures): Features = {
     val withStringFeatures = features.stringFeatures
       .foldLeft(Features.newBuilder()) {
         case (bldr, (nodeName: String, stringFeature: String)) =>
```

Comment:
out of curiosity, what does bldr stand for?

Reply:
builder.  Changing it to that, so it's not confusing :)